openapi/azure.sqldb/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerinax"
 name = "azure.sqldb"
-version = "1.0.0-SNAPSHOT"
+version = "1.0.1"
```

Comment:
This connector is not released yet.
So the version should be `1.0.0`

Reply:
Sure. I will update as that.

Reply:
Updated on all places @LakshanSS